rotkehlchen/inquirer.py

```diff
@@ -238,6 +238,10 @@ def __new__(
     def inject_ethereum(ethereum: 'EthereumManager') -> None:
         Inquirer()._ethereum = ethereum
 
+    @staticmethod
+    def is_cache_valid(cache: CachedPriceEntry) -> bool:
```

Comment:
Since you went ahead and made a function then why not just copy all logic inside the function?

Pass the `cache_key` here and add both the cache retrieval and the check for `None` in the function itself.